src/main/java/Echo.java

```diff
@@ -0,0 +1,175 @@
+                }
+
+                String deadlineInput = input.substring("deadline".length()).trim();
+                if (!deadlineInput.contains("/by ")) {
+                    System.out.println("Deadline time is missing. Please use '/by' to specify the deadline.");
+                    break;
+                }
+
+                String[] deadlineParts = deadlineInput.split(" /by ", 2);
+                if (count == list.length) {
+                    list = Arrays.copyOf(list, count * 2);
+                }
+
+                list[count] = new Deadline(deadlineParts[0], deadlineParts[1]);
+                count++;
+                System.out.println(break_line);
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  [D][ ] " + deadlineParts[0] + " (by: " + deadlineParts[1] + ")");
+                System.out.println("Now you have " + count + " tasks in the list.");
+                System.out.println(break_line);
+                break;
+
+            case "event":
+                if (!input.toLowerCase().startsWith("event")) {
+                    // Handle this as a generic task or provide a specific message
+                    System.out.println("Command not recognized. Please use 'event' for event tasks.");
+                    break;
+                }
+
+                String eventInput = input.substring("event".length()).trim();
+                if (!eventInput.contains("/from ") || !eventInput.contains("/to ")) {
+                    System.out.println("Event time details are missing. Please use '/from' and '/to' to specify the event timing.");
+                    break;
+                }
+
+                String[] eventParts = eventInput.split(" /from ", 2);
+                String[] timeParts = eventParts[1].split(" /to ", 2);
+                if (count == list.length) {
+                    list = Arrays.copyOf(list, count * 2);
+                }
+
+                list[count] = new Event(eventParts[0], timeParts[0], timeParts[1]);
+                count++;
+                System.out.println(break_line);
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  [E][ ] " + eventParts[0] + " (from: " + timeParts[0] + " to: " + timeParts[1] + ")");
+                System.out.println("Now you have " + count + " tasks in the list.");
+                System.out.println(break_line);
+                break;
+
+            case "clearlist":
+                Arrays.fill(list, null); // Clearing the list
+                count = 0; // Resetting the count
+                System.out.println(break_line);
+                System.out.println("The List has been cleared!");
+                System.out.println(break_line);
+                break;
+
+            case "bye":
+                scanner.close();
+                System.out.println("Kkkkkk thanks bye ");
+                return;
+
+            case "mark":
```

Comment:
You could consider refactoring this code to prevent deep nesting and to adhere to this code quality [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting)